src/foundry/foundry.js/applications/sidebarTabs/combatTracker.d.ts

```diff
@@ -2,184 +2,188 @@
 // eslint-disable-next-line
 // @ts-nocheck
 
-/**
- * The combat and turn order tracker tab
- */
-declare class CombatTracker extends SidebarTab<CombatTracker.Options> {
-  /**
-   * Record a reference to the currently highlighted Token
-   * @defaultValue `null`
-   */
-  protected _highlighted: Token | null;
-
-  /**
-   * Record the currently tracked Combat encounter
-   * @defaultValue `null`
-   */
-  combat: Combat | null;
+import { ConfiguredDocumentClass } from '../../../../types/helperTypes';
 
+declare global {
   /**
-   * @override
+   * The combat and turn order tracker tab
    */
-  static get defaultOptions(): CombatTracker.Options;
-
-  /**
-   * @override
-   */
-  createPopout(): CombatTracker;
-
-  /**
-   * Initialize the combat tracker to display a specific combat encounter.
-   * If no encounter is provided, the tracker will be initialized with the first encounter in the viewed scene.
-   * @param combat - The combat encounter to initialize
-   *                 (default: `null`)
-   * @param render - Whether to re-render the sidebar after initialization
-   *                 (default: `true`)
-   */
-  initialize({ combat, render }?: { combat?: Combat | null; render?: boolean }): void;
-
-  /**
-   * Scroll the combat log container to ensure the current Combatant turn is centered vertically
-   */
-  scrollToTurn(): void;
+  class CombatTracker extends SidebarTab<CombatTracker.Options> {
+    /**
+     * Record a reference to the currently highlighted Token
+     * @defaultValue `null`
+     */
+    protected _highlighted: Token | null;
 
-  /**
-   * @override
-   */
-  getData(options?: Application.RenderOptions): Promise<CombatTracker.Data>;
+    /**
+     * Record the currently tracked Combat encounter
+     * @defaultValue `null`
+     */
+    combat: StoredDocument<InstanceType<ConfiguredDocumentClass<typeof Combat>>> | null;
```

Comment:
I think this variable has been renamed to `viewed`...? Also I'm not sure I would do it like this here. What's preventing modules / systems to view a combat in the combat tracker that is only temporary?

Reply:
Yes, it has been renamed. Have updated this.

Viewing a combat in the combat tracker that is only temporary would require a massive amount of code, because most event handlers rely on the combat ID and `game.combats` (and also `getData` and many more).

But I have no real prejudice here. If you think that there might be someone who will do this, I surely can remove the stored here without being annoyed or anything like this.